frontend/src/components/PlacementRuleDeprecationAlert.tsx

```diff
@@ -1,15 +1,19 @@
 /* Copyright Contributors to the Open Cluster Management project */
 
-import { Alert } from '@patternfly/react-core'
+import { Alert, TextContent } from '@patternfly/react-core'
 import React from 'react'
 import { useTranslation } from '../lib/acm-i18next'
+import { viewDocumentation, DOC_LINKS } from '../lib/doc-util'
 
 const PlacementRuleDeprecationAlert = () => {
   const { t } = useTranslation()
   return (
     <React.Fragment>
       <Alert variant="info" isInline title={t('Placement rule deprecation')} style={{ marginBottom: '10px' }}>
-        <p>{t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}</p>
+        <TextContent>
+          {t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}
```

Comment:
@mprahl 

I suggest the following updates:

Change PlacementRule is deprecated and will not receive updates or fixes.

TO 

Placement rule is deprecated and will not receive updates or fixes.
OR
PlacementRule resource is deprecated and will not receive updates or fixes.

Not sure if we can move Best practice to the next line and bold "Best practice:"